components/layout/display_list/builder.rs

```diff
@@ -2842,11 +2829,7 @@ impl InlineFlowDisplayListBuilding for InlineFlow {
         for fragment in self.fragments.fragments.iter_mut() {
             state.containing_block_clipping_and_scrolling = previous_cb_clipping_and_scrolling;
 
-            if establishes_containing_block_for_absolute(
-                StackingContextCollectionFlags::empty(),
-                fragment.style.get_box().position,
-                false,
-            ) {
+            if fragment.style.get_box().position != StylePosition::Static {
```

Comment:
And same here?

This looks wrong btw, an inline can also establish a reference frame.

Reply:
Want me to file a follow-up issue? This is just the same code as before right?